packages/components/src/components/files/FileTree.tsx

```diff
@@ -32,7 +32,8 @@ const customStyle = {
                 display: 'block',
             },
             activeLink: {
-                background: 'white',
+                background: 'linear-gradient(90deg, rgba(211,211,211,1) 20%, rgba(255,255,255,1) 80%)',
+                borderRadius: '5px'
```

Comment:
Did you run this gradient display past MattY to get his thoughts on it?

Reply:
MattY got back to me and suggested as an alternative, rather than changing the background, we could change the font color of the active node instead.

The styling when it comes to customStyle gets shifty due to how the container nesting will override styling, see [this](https://github.com/storybookjs/react-treebeard/issues/57) thread.

However, it's possible and easy (I didn't have to imitate the commenter's more complicated code to keep track of what is active) to change the color of the highlighted node + file icon. The behavior of it is luckily pretty stable and reliable! It always highlights the last node clicked.
![Screen Shot 2020-07-30 at 3 42 47 PM](https://user-images.githubusercontent.com/18537731/88981682-5cebf780-d27b-11ea-9ae7-6615b3e74ac2.png)

_Unfortunately_ the arrow is outside of what we can control in the Header. The div the arrow is a parent of our eldest Header span, with the className 'filetree-checkbox-container'. I've looked around but couldn't find a way to highlight the arrow only on activation. I've pinged MattY about it, just to get his thoughts.

Reply:
I like this color highlighting for the single node selection case. It doesn't bother me that the arrow isn't highlighted as long as the icon and text are. I would bother messing with the arrow part. Thanks.

Reply:
Note: It was decided a `allowMultiSelect` prop of `true` triggers the single-node highlighting, but `false` will have no highlighting for now.